pkg/asset/installconfig/openstack/validation/platform_test.go

```diff
@@ -53,6 +53,7 @@ func withControlPlanePortSubnets(subnetCIDR, allocationPoolStart, allocationPool
 			AllocationPools: []subnets.AllocationPool{
 				{Start: allocationPoolStart, End: allocationPoolEnd},
 			},
+			IPVersion: 4,
```

Comment:
Can you also add a test for single stack IPv6?

Reply:
There is a test case for single stack IPv6 [here](https://github.com/openshift/installer/pull/8925/files#diff-5225a14336843fbe254ed6b44298cb320f23b931b30df8ebb60e21d2e613cec9R595-R625).

Also this function `withControlPlanePortSubnets` only runs during VIPs allocation pool [validation](https://github.com/openshift/installer/pull/8925/files#diff-5225a14336843fbe254ed6b44298cb320f23b931b30df8ebb60e21d2e613cec9R230-R259), which is not performed when using dual-stack or IPv6 VIPs as they are provided by the user.

Do you have any specific test case in mind?

Reply:
No, all good then